webhooks.go

```diff
@@ -0,0 +1,33 @@
+package slack
+
+import (
+	"github.com/pkg/errors"
+	"net/http"
+	"bytes"
+	"encoding/json"
+)
+
+type WebhookMessage struct {
```

Comment:
do we need another type here or will the standard type Message (or one of its variants) work?

Reply:
The slack documentation indicates that only these fields are used. It would be confusing/bad design to use a struct with fields that are unused by the request.

Reply:
The solution to this would be to use interfaces for inputs. That would require custom serialization.